test/sslapitest.c

```diff
@@ -13093,6 +13093,24 @@ static int test_quic_tls_early_data(void)
 }
 #endif /* !defined(OSSL_NO_USABLE_TLS1_3) */
 
+#ifdef OPENSSL_SYS_WINDOWS
+static int test_load_winstore(void)
+{
+    int ret = 0;
+    SSL_CTX *ctx = SSL_CTX_new_ex(libctx, NULL, TLS_client_method());
+
+    if (!TEST_ptr(ctx))
+        goto err;
+    if (!TEST_int_eq(SSL_CTX_load_verify_store(ctx, "org.openssl.winstore:"), 1))
+        goto err;
+    ret = 1;
+err:
+    SSL_CTX_free(ctx);
+    return ret;
+}
+
+#endif
```

Comment:
I tried taking out the `ifdef OPENSSL_SYS_WINDOWS` and running this on Linux. The test passed. I don't think this does what you think it does. This doesn't actually load the winstore. It just stores away the URI in an internal stack ready for later use.